src/status_im/contexts/wallet/create_account/new_keypair/keypair_name/view.cljs

```diff
@@ -42,4 +50,20 @@
             :label           (i18n/label :t/keypair-name)
             :char-limit      keypair-name-max-length
             :auto-focus      true
-            :on-change-text  #(reset! keypair-name %)}]]]))))
+            :on-change-text  (fn [value]
+                               (reset! keypair-name value)
+                               (cond
+                                 (> (count value) keypair-name-max-length) (set-error? 1)
```

Comment:
imo it would be nice to use keywords for the errors over integers, it is more descriptive that way

Reply:
additionally, `set-error?` shouldn't end with a question mark